kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -495,19 +507,15 @@ public String getTopicToDataset(String topicName) {
    * @return a {@link SchemaConverter} for BigQuery.
    */
   public SchemaConverter<Schema> getSchemaConverter() {
-    return getBoolean(INCLUDE_KAFKA_DATA_CONFIG)
-        ? new KafkaDataBQSchemaConverter(getBoolean(ALL_BQ_FIELDS_NULLABLE_CONFIG))
-        : new BigQuerySchemaConverter(getBoolean(ALL_BQ_FIELDS_NULLABLE_CONFIG));
+    return new BigQuerySchemaConverter(getBoolean(ALL_BQ_FIELDS_NULLABLE_CONFIG));
```

Comment:
Could you explain a little on this change? 😃

Reply:
We used to have a `KafkaDataBQSchemaConverter` class to convert schema with kafka data. But that implementation caused difficulty for us to make changes to include kafka key fields. So I removed that class thus make changes here accordingly.

Reply:
Aha, I see!